platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/gradle_directories.adoc

```diff
@@ -73,7 +73,7 @@ The project root directory contains all source files from your project.
 
 It also contains files and directories Gradle generates, such as `.gradle` and `build`.
 
-While `.gradle` is usually checked into source control, the `build` directory contains the output of your builds as well as transient files Gradle uses to support features like incremental builds.
+While `gradle` is usually checked into source control, the `build` directory contains the output of your builds as well as transient files Gradle uses to support features like incremental builds.
```

Comment:
@lkasso  While it's definitely a correct change, it would be nice to introduce an explanation for `gradle` dir somewhere above.

Reply:
Yeah, it's confusing. But the previous statement is also correct: `dot-gradle` and `build` are generated. `gradle` is not.

Any suggestions?

> A directory, containing Gradle Wrapper files and Gradle Version Catalog?

Reply:
> But the previous statement is also correct:

Yes.

> Any suggestions?

It would be nice to add it to the sentence above: `it also contains ... %here... and files and directories Gradle generates...`

> A directory, containing Gradle Wrapper files and Gradle Version Catalog?

It's not just that. It can also contain dependency verification and other stuff. "Gradle configuration folder"?

Reply:
#30068